### DISCUSSION TITLE
But shut down in a longer working time

### DISCUSSION THREAD

**Discussion by Sirius3331** [eritislami/evobot#878] [Q&A] [answered]
Hello guys,

I have a little problem to run my bot for a much longer time. It runs about 30 minutes and it goes offline. I am running the bot on replit. The other functions are working perfectly but it's not so nice that I have to start the bot again and again. Do you know how to solve it?

Thanks in advance

Kind regards
Sirius3331

**Comment by Xe-Xo** [comment]
Sounds like you are using the Starter account on repl.it. It automatically shutdowns after a period of time.
You can try upgrading to Hacker -> https://replit.com/site/pricing

**Comment by wolf-yuan-6115** [comment, marked as answer]
Or setup uptimerobot to ping it every 5 minutes

**Reply by Sirius3331** [reply]
Thank you for the help. I found out how to do it and It's working.

**Comment by Sirius3331** [comment]
Thank you for a reply. At first I'm sorry that I am respond so late. Your two are good but I don't want to waste money. That mean that I tried out with the uptimebot but I don't really know where I have to put in the code's. I stepped into the programming world a week ago and that's why my experience is at the time not so good. Can you help me a little bit which uptimerbot should I use and where I have to put the codes in?
